ndspy/bmg.py

```diff
@@ -385,17 +410,22 @@ def save(self, encoding):
             data.extend(self.data)
             return data
 
-        def __repr__(self):
+        def __repr__(self) -> str:
             return f'{type(self).__name__}({self.type!r}, {self.data!r})'
 
-        def __str__(self):
+        def __str__(self) -> str:
             return f'[{self.type}:{self.data.hex()}]'
 
-    info = 0
-    stringParts = None
-    isNull = False
+    info: bytes
+    stringParts: list[str]
```

Comment:
I almost forgot, this is actually a `list[str | Escape]`, not just of `str` (this also applies to the corresponding `__init__` argument)